src/objects/api/utils.py

```diff
@@ -43,3 +43,22 @@ def display_choice_values_for_help_text(choices: DjangoChoices) -> str:
         items.append(item)
 
     return "\n".join(items)
+
+
+def merge_patch(target: Any, patch: Any) -> Dict[str, Any]:
```

Comment:
target and patch should always be JSON values, so maybe the type hints can be more specific?

Reply:
This is true for the first call to `merge_patch`, but nested calls doesn't necessarily have dicts as values.

In fact merge Patch can also be applied for any kind of data (you could have arrays for example, even if it doesn't really make sense to patch an array with another one).

Reply:
What I had in mind is something like what we have in OF, where we define these types:

```python
JSONPrimitive = Union[str, int, None, float, bool]

JSONValue = Union[JSONPrimitive, "JSONObject", List["JSONValue"]]
```
And then I thought maybe instead of `Any` we could use `JSONValue`?

Reply:
Thought about that as well, but for such a small use case decided to go with `Any`. But I think this is still a good idea as typing JSON arguments is common so this could be reused.